WordPress/Classes/ViewRelated/Reader/ReaderViewController.m

```diff
@@ -15,7 +15,8 @@
 #import "WordPress-Swift.h"
 
 @interface ReaderViewController () <UIViewControllerRestoration>
-@property (nonatomic, strong) ReaderPostsViewController *postsViewController;
+//@property (nonatomic, strong) ReaderPostsViewController *postsViewController;
+@property (nonatomic, strong) ReaderStreamViewController *postsViewController;
```

Comment:
Was the official switch from `ReaderPostsViewController` > `ReaderStreamViewController` intended?